code/datums/brain_damage/special.dm

```diff
@@ -222,7 +222,8 @@
 	QDEL_NULL(psychotic_brawling)
 
 /datum/brain_trauma/special/psychotic_brawling/bath_salts
-	name = "Chemical Violent Psychosis"
+	name = "Chemical Violent Psychosis" //note that this DOESN'T change scan_desc, so this won't change what health analyzers refer to this brain trauma as (in comparison to non-chemical violent psychosis)
```

Comment:
it'd probably make sense to modify scan_desc as well

Reply:
I would, but then the scanner would say that the brain trauma from bath salts is "permanent chemically induced violent psychosis" or something like that.

Reply:
Good point. On second thought it actually would make more sense to hide it completely, since it's more of a status effect that uses trauma code for convenience. I wonder why i never made the scan_desc optional.